app/views/practice.py

```diff
@@ -69,6 +69,10 @@ def pull_team_stats(team_id: int, sheet: Sheet, match: Match) -> dict:
 def get_team_model(team_id: int, difficulty: int) -> PracticeTeam:
     """Return the team simulation model"""
     sess = get_active_session()
+    if sess is None:
+        sess = Session.query.filter(Session.active is True).first()
```

Comment:
```suggestion
        sess = Session.query.filter(Session.active == True).first()
```
I am not sure how SQLAlchemy would convert the filter condition to SQL when you use `is` rather than `==` (therefore `__eq__`). Might be a subtle bug that always falls back to the next case where `sess is None`

Reply:
@glifchits  I believe you are right need to use == True. 
https://stackoverflow.com/questions/18998010/flake8-complains-on-boolean-comparison-in-filter-clause

Reply:
Fixed in 5cb93f5dd723a5ea806df95bd4c91172776f8b2f